class.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'module'
+class Mylist
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] `MyList` class needs to have a #each method that yields all members of @list in succession.
![image](https://user-images.githubusercontent.com/62705947/191758185-b4ca146d-5429-403c-a4e8-497a0d8245e4.png)

Reply:
@GzimAsani Thank you for reviewing my project, I will make the necessary corrections as suggested